dev/run-tests-jenkins.py

```diff
@@ -165,12 +165,6 @@ def main():
     if "test-maven" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_TOOL"] = "maven"
     # Switch the Hadoop profile based on the PR title:
```

Comment:
Yep. This is not related to the SparkPullRequestBuilder failures. 
SparkPullRequestBuilder is due to the other file in this PR.